src/api/profile.ts

```diff
@@ -38,3 +38,37 @@ export async function deleteProfile(opts?: AxiosRequestConfig) {
 
   return deleteOK;
 }
+
+/**
+ * Function to get customer account ID based on phone number
+ *
+ * @param phoneNumber phone number with prefix
+ *
+ * @returns customer account ID as string if request is success
+ *
+ * Will return one the following errors if request unsuccessful:
+ * -  Error 404 with object {"error": "cannot find user from phone number: \"{phoneNumber}\""}
+ * -  Error 400 with object {"error": "Invalid phone number as per country code"}
+ * -  other than those 2 errors mentioned earlier, returns undefined.
+ */
+export const getCustomerAccountId = async (phoneNumber: string) => {
```

Comment:
Could you type the return type from this function? A little hard to see just by reading the code.